README.md

```diff
@@ -89,8 +89,11 @@ Create an OpenTofu variables file to define the required infrastructure, e.g.:
     cluster_subnet = "some_subnet" # *
     key_pair = "my_key" # *
     control_node_flavor = "some_flavor_name"
-    login_nodes = {
-        login-0: "login_flavor_name"
+    login = {
+        interactive = {
```

Comment:
Is `interactive` significant? Would you ever need more than one group here?

If you'd only ever have one group, then perhaps this could be handled down in the module rather than the slightly complicated data structure here. If you could have more than one group, maybe a few lines of docs or a comment to explain the data structure would be helpful?